### PR TITLE
Bring Jason's fork into develop

### PR DESCRIPTION
**Pull request by ProjectSlxsh**
Long story short jasonlovesdoggo has a lot of features that could be implemented and I am not going to sit here and act like i know what I am doing but i will most a photo of him saying it 
image

### CODE REVIEW

**Comment by JasonLovesDoggo** [comment]
I will have to revert the readme though

**Comment by ProjectSlxsh** [comment]
Wow that's more than I thought, you don't have to just wanted to make a pr so you didn't have to

**Comment by JasonLovesDoggo** [comment]
started transition

**Comment by JasonLovesDoggo** [comment]
see #490

File: main.py [hunk 1]
```diff
@@ -1,115 +1,72 @@
...
+╚═╝  ╚═╝╚══════╝╚═════╝ ╚═════╝ ╚═╝   ╚═╝         ╚═══╝  ╚═╝╚═════╝ ╚══════╝ ╚═════╝     ╚═╝     ╚═╝╚═╝  ╚═╝╚═╝  ╚═╝╚══════╝╚═╝  ╚═╝
+"""
 print(banner)
-time.sleep(0.5)
-
+load_dotenv()
+# base code by elebumm
 print_markdown(
-    "### Thanks for using this tool! [Feel free to contribute to this project on GitHub!](https://lewismenelaws.com) If you have any questions, feel free to reach out to me on Twitter or submit a GitHub issue."
+    "### Thanks for using this tool! 😊 Feel free to contribute to this project on GitHub! (JasonLovesDoggo/RedditVideoMakerBot). If you have any questions, feel free to reach out to me on Twitter @JasonLovesDoggo or submit a GitHub issue."
```
Comment (CallumIO) [inline main.py:26]: This should be reverted back to what it was originally

Comment (JasonLovesDoggo) [reply main.py:26]: yeah

**Comment by FreebieII** [comment]
Works well for me. Final video was created with no issues.

**Review by JasonLovesDoggo — APPROVED** [review]
everything works for me.

**Comment by JasonLovesDoggo** [comment]
Will merge at 8:55 a.m. EST

**Comment by JasonLovesDoggo** [comment]
Merging now.

**Pull request merged**